Modules/admin/admin_model.php

```diff
@@ -617,8 +617,8 @@ public function is_Pi() {
     /**
      * get an array of raspberry pi properites
      *
-     * @see: SoC 'hw' now not required - https://github.com/emoncms/emoncms/issues/1364
-     * @return array has keys [hw,]rev,sn,model     * @return array has keys hw,rev,sn,model
```

Comment:
The `@return` was duplicated and isn't helpful.

`now not required` is completely different to `not used ever`